src/naive/datetime/tests.rs

```diff
@@ -4,6 +4,91 @@ use crate::NaiveDate;
 use crate::{Datelike, FixedOffset, Utc};
 use std::i64;
 
+#[test]
+fn test_datetime_from_timestamp_millis() {
+    let timestamp_millis: i64 = 1662921288000; //Sunday, September 11, 2022 6:34:48 PM
```

Comment:
Okay now there's kind of a lot of duplication, though. Can we extract the test data out into a slice and loop over that?

Also doesn't seem useful to assert `is_some()` if the next line is going to `unwrap()` anyway?

Reply:
Good point - used a map with a loop, and removed the assert some